census-us/Readme.md

```diff
@@ -86,13 +89,5 @@ REVERSE GEOGRAPHY CODER:
 
 Please create the code-env according to the definition provided with the plugin
 
-Otherwise, without code-envs:
-This plugin is using the Requests python Lib. For accessing the US Census servers Requests need some security addons that can be found in the request[security] lib. Since Requests is used by DSS, it's proposed to install the following dependencies separately:
-
-- idna<2.6,>=2.5
-- cryptography>=1.3.4
-- pyopenssl>=0.14
-=> These dependencies can be tricky to install on Mac OS and only due to pip, not the plugin. Mind: Mac OS not officially supported by Dataiku : https://doc.dataiku.com/dss/latest/installation/index.html
```

Comment:
bigger idea: do we have any way of like creating Conda environments FROM DSS to bring in all necessary dependencies?